tests/unit/s2n_tls13_cert_verify_test.c

```diff
@@ -234,5 +234,121 @@ int main(int argc, char **argv)
         run_tests(&test_cases[i], S2N_SERVER);
     }
 
+    /* Self-talk: Ensure that the signature algorithm used to sign the CertificateVerify message
+     * is validated against the certificate type
+     */
+    if (s2n_is_tls13_fully_supported()) {
+        struct s2n_tls13_cert_verify_test sha256_test_cases[] = {
```

Comment:
Nit: This test is a little hard to follow. Maybe because sha256_test_cases and test_sha256_sig_algs are such similar names. Since technically the former is the test server params and the latter is the test client params maybe you could distinguish them that way?